.babelrc

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
I see you removed the inlined version.  As long as it's in one place I'm happy :)